src/boards/include/boards/solderparty_rp2040_stamp_round_carrier.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2022 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+//
+//------------------------------------------------------------------------------------------
+// Board definition for the Solder Party RP2040 Stamp Round Carrier
+//
+// This header may be included by other board headers as "boards/solderparty_rp2040_stamp_round_carrier.h"
+
+#ifndef _BOARDS_SOLDERPARTY_RP2040_STAMP_ROUND_CARRIER_H
+#define _BOARDS_SOLDERPARTY_RP2040_STAMP_ROUND_CARRIER_H
+
+// For board detection
+#define SOLDERPARTY_RP2040_STAMP_ROUND_CARRIER
+
+// --- User LED ---
+#ifndef PICO_DEFAULT_LED_PIN
+#define PICO_DEFAULT_LED_PIN   25
+#endif
+
+// --- UART ---
+#ifndef PICO_DEFAULT_UART_TX_PIN
+#define PICO_DEFAULT_UART_TX_PIN 16
+#endif
+#ifndef PICO_DEFAULT_UART_RX_PIN
+#define PICO_DEFAULT_UART_RX_PIN 17
+#endif
+
+// --- Neopixel ---
+#ifndef PICO_DEFAULT_WS2812_PIN
+#define PICO_DEFAULT_WS2812_PIN 24
```

Comment:
Looking at the product-page you mentioned, I guess the on-board NeoPixel doesn't stop working when the Stamp is on the Carrier Board, so perhaps the  Carrier Board NeoPixel pin should get its own `#define`, rather than overriding the `PICO_DEFAULT_WS2812_PIN` from `solderparty_rp2040_stamp.h`? :shrug:

Reply:
I thought about it, but I think I'd rather have the Carrier NeoPixel ring take priority over the single NeoPixel on the Stamp 😅

Reply:
Okay, so perhaps you want to add an additional define to the carrier's header-file with the definition of the stamp's neopixel pin, so that code written to target the carrier can write to both (defined) neopixel pins? :slightly_smiling_face: